client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -35,9 +36,17 @@ export const subscriptionsFromApi = apiResponse => {
 		return [];
 	}
 
-	return map( apiResponse.subscriptions, subscription =>
-		omit( subscription, 'meta' )
-	);
+	return map( apiResponse.subscriptions, subscription => {
+		return omitBy( {
+			ID: Number( subscription.ID ),
+			URL: subscription.URL,
+			blog_ID: toValidId( subscription.blog_ID ),
```

Comment:
do you think we should change everything to blogId/feedId? Whats your feeling on that in general?

Right now in my head I'm thinking it could be nice if we modified everything to map to feedId/siteId in the data-layer